src/main/java/com/springml/salesforce/wave/model/DescribeSObjectResult.java

```diff
@@ -0,0 +1,149 @@
+package com.springml.salesforce.wave.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+import java.io.Serializable;
+import java.util.List;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class DescribeSObjectResult implements Serializable {
```

Comment:
just curious, is this bean class generated or hand-written?

Reply:
The private properties were hand-written. The getters and setters were generated.